File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.py
def solution(my_str, n):
    answer = []
    a = ''
    for i in range(len(my_str)):
        a = a+my_str[i]
        if len(a)==n or i == len(my_str)-1:
            answer.append(a)
            a = ''
            
    return answer

File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.java
import java.util.List;
import java.util.ArrayList;

class Solution {
    public String[] solution(String my_str, int n) {
        List<String> answer = new ArrayList<>();
        StringBuilder word = new StringBuilder();
        char[] str = my_str.toCharArray();
        for (int i = 0; i < str.length; i++) {
            word.append(str[i]);
            if (word.length() == n || i == str.length - 1) {
                answer.add(word.toString());
                word.setLength(0);
            }
        }
        return answer.toArray(new String[0]);
    }
}